magento/cart2quote/2017-02-01.yaml

```diff
@@ -0,0 +1,13 @@
+
+title:     Remote Code Execution in Qquoteadv/controllers/DownloadController.php
+link:      https://cart2quote.zendesk.com/hc/en-us/articles/115000616303--FIXED-Security-Vulnerability-in-downloadCustomOptionAction
+cve:       ~
+branches:
+    4:
+        time:     2017-02-01 10:45:00
+        versions: ['>=4.1.6', '<5.0'
+    5:
+        time:     2017-02-01 10:45:00
+        versions: ['>=5.0.1', '<5.4.4']
+reference: composer://cart2quote/cart2quote
```

Comment:
according to their installation doc, the package is name ``cart2quote/module-quotation`` or ``cart2quote/module-quotation-encoded`` btw (depending on the version being used).
Using the wrong package name here would make the checker tool unable to detect the usage of a vulnerable version